### PR TITLE
Setup API calls to Spotify for authentication

### PR DESCRIPTION
**Pull request by jimwelagu**

### CODE REVIEW

File: app/src/main/java/com/example/musicroulette/utils/SpotifyUtils.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.example.musicroulette.utils;
+
+import android.net.Uri;
+
+import okhttp3.MultipartBody;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+
+public class SpotifyUtils {
```
Comment (buitim) [inline app/src/main/java/com/example/musicroulette/utils/SpotifyUtils.java:9]: You're going to have to show me how you figured out how to build this haha

**Review by buitim — APPROVED** [review]
Everything makes sense and looks good. Was it tested yet? And I'm not that great with the backend network communication stuff. Should I try working on the interface?

Approved.

File: app/src/main/java/com/example/musicroulette/utils/SpotifyUtils.java (hunk 1 above)
Comment (jimwelagu) [reply app/src/main/java/com/example/musicroulette/utils/SpotifyUtils.java:9]: Yeah lets meet up sometime this week (before friday) if everyone has time so I can explain what I did.

**Comment by jimwelagu** [comment]
> Everything makes sense and looks good. Was it tested yet? And I'm not that great with the backend network communication stuff. Should I try working on the interface?
> 
> Approved.

Yes I made a AsyncTask to test the network call. But, the response I'm getting is in HTML page for the user to login with their Spotify account. I was doing some research on this and we might need to use something like Retrofit to handle the HTML response.
https://mobikul.com/handle-html-response-from-http-request-using-retrofit/

**Pull request merged**